tests/test_physics.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from deepqmc.physics import local_potential, nonlocal_potential
+
+
+@pytest.mark.parametrize('pp_type', [None, 'bfd', 'ccECP'])
+@pytest.mark.parametrize('name', ['LiH', 'C', 'ScO'])
+class TestPhysics:
+    def test_pseudo_potentials(self, helpers, name, pp_type, ndarrays_regression):
+        mol = helpers.mol(name, pp_type)
+        params, state, paulinet, rs = helpers.create_paulinet(
+            hamil=helpers.hamil(mol, elec_std=0.45)
+        )
+        print(rs)
```

Comment:
Could you remove this `print` statement?

Reply:
Sorry about that, I've removed it in dc4d99c430a903705e9a35fe97a084e6023353c2 .